webapp/src/index.tsx

```diff
@@ -306,6 +324,23 @@ export default class Plugin {
                     return {};
                 }
                 return {error: {message: 'You are not connected to a call in the current channel.'}};
+            case 'end':
+                if (voiceConnectedUsersInChannel(store.getState(), args.channel_id)?.length === 0) {
+                    return {error: {message: 'No ongoing call in the channel.'}};
+                }
+
+                if (!isCurrentUserSystemAdmin(store.getState()) &&
+                    getCurrentUserId(store.getState()) !== voiceChannelCallCreatorID(store.getState(), args.channel_id)) {
+                    return {error: {message: 'You do not have permission to end the call. Please ask the call creator to end call.'}};
```

Comment:
```suggestion
                    return {error: {message: 'You don't have permission to end the call. Please ask the call creator to end the call.'}};
```

Reply:
@streamer45, can we replace "call creator" with "host" or "owner"?

Reply:
We can, I guess it's more of a product question at this point as we don't really have a concept of "hosting a call". The "creator" is the one that originally started the call. I am okay with owner. /cc @itao

Reply:
I'm onboard with `owner` as it's implicitly transferrable, whereas `creator` can only be one person evern.

Reply:
Perfect. I'll update, thanks!